SETUP.md

```diff
@@ -0,0 +1,166 @@
+# Cloning manubot-rootstock to create a new manuscript
+
+The process to create a new Manubot manuscript is a bit challenging, because it requires a few steps that are difficult to automate.
+However, you will only have to perform these steps once for each manuscript.
+These steps should be performed in a terminal, starting in the directory where you want the manuscript folder be created.
+
+## Configuration
+
+First, you must configure two environment variables (`OWNER` and `REPO`).
+These variables specify the GitHub repository for the manuscript (i.e. `https://github.com/OWNER/REPO`).
+
+```sh
+# GitHub account (change from greenelab)
+OWNER=greenelab
+# Repository name (change from manubot-rootstock)
+REPO=manubot-rootstock
+```
+
+The following commands assume you have SSH key access configured for the `OWNER` GitHub account.
+If this is not the case, see GitHub's guidelines for [generating](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) a new SSH key and [adding it](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/) to your account.
+
+## Create repository
+
+Next you must clone `greenelab/manubot-rootstock` and configure its branches and remotes:
+
+```sh
+# Clone greenelab/manubot-rootstock
+git clone https://github.com/greenelab/manubot-rootstock.git $REPO
+cd $REPO
+
+# Configure remotes and branches
+git remote add rootstock https://github.com/greenelab/manubot-rootstock.git
+git checkout gh-pages
+git checkout output
+git checkout master
+git remote set-url origin git@github.com:$OWNER/$REPO.git
+```
+
+Next, you must manually create an empty GitHub repository at https://github.com/new.
+Make sure to use the same "Owner" and "Repository name" specified above.
+Do not initialize the repository, other than optionally adding a Description.
+Next, push your cloned manuscript:
+
+```sh
+git push --set-upstream origin master
+git push --set-upstream origin gh-pages
+git push --set-upstream origin output
+```
+
+## Continuous integration
+
+Now you must manually enable Travis CI for the new repository at https://travis-ci.org.
```

Comment:
Could link to https://docs.travis-ci.com/user/getting-started/ for users who aren't already using Travis CI.

Reply:
I think that link may be more confusing than the landing page for users who haven't authorized a GitHub account at https://travis-ci.org.